server/api/routes/meeting.cjs

```diff
@@ -93,6 +94,81 @@ router.delete('/:meetingId', requireToken, isAdmin, async (req, res, next) => {
     next(err);
   }
 });
+router.post('/active/messages', requireToken, async (req, res, next) => {
+  try {
+    let correctRecip;
+    const meeting = await Meeting.findOne({
+      where: {
+        isClosed: false,
+        meetingStatus: 'confirmed',
+        [Op.or]: [{ userId: req.user.id }, { buddyId: req.user.id }],
+      },
+    });
+    if (meeting === undefined) {
+      res.status(404).send('User is not in a meeting that is confirmed');
+    }
+    console.log(typeof req.user.id);
```

Comment:
remove comment if it's not necessary and if it is, write text that describes what this is logging